nntrainer/tensor/cl_operations/blas_kernels_fp16.cpp

```diff
@@ -333,12 +335,18 @@ void addition_cl(const __fp16 *input, __fp16 *res, unsigned int size) {
     }
 
     result =
-      kernel_addition_fp16_ptr->SetKernelArguments(2, &size, sizeof(int));
+      kernel_addition_fp16_ptr->SetKernelArguments(2, &size_input, sizeof(int));
     if (!result) {
       break;
     }
 
-    const int work_groups_count[3] = {(int)size, 1, 1};
+    result =
+      kernel_addition_fp16_ptr->SetKernelArguments(3, &size_res, sizeof(int));
+    if (!result) {
+      break;
+    }
+
+    const int work_groups_count[3] = {(int)size_res, 1, 1};
     const int work_group_size[3] = {32, 32, 1}; // test-value
```

Comment:
This is simple question.
How can we optimize this work_group_size? Do you have any idea or plan on this?

Reply:
We are thinking on this. Will let you know. Thanks for reviewing.